### DISCUSSION TITLE
"Fixed YOURSELF Colour" - Your colour to be always the same (in all games)

### DISCUSSION THREAD

**Discussion by VanderLeal** [bafolts/terraforming-mars#1313] [Ideas]
(enhancement request)

Possibly Cookie based.

You see always the same color for your games.
You would always see yourself as the "RED" color, regardless how others see you.
Each person has their own view of the board. Colours are only cosmetic but very important to the interface. 
A bad colour decreases the user experience.

Optional, although imho is would be better and not harder to implement:
Make the "yourself colour" unpickable. Example: Purple.
This way the colours would still work the very same way, except each person would see yourself in the "yourself colour". When I share a screenshot with other players, the "non-yourself" colours would still match.

PS: Thank you very much for all the effort. This project and the results this application has so far is quite impressive, for such complex game. Kudos!

PS'': this is my first time interacting here, if I'm not following any protocol, please accept my apologies.

**Comment by chosta** [comment]
Hi, 
first of all no need to apologise for anything :)
The project exists not only because of the hard-working devs but also because of patient players that give valuable feedback and ideas like you.
We are currently contemplating a change in the color system meaning we want to change the base colors to a palette that is more appealing to the eye and less problematic for the UI (contrast issues with black at the moment).
Regarding your favourite color idea - yes, cookie might be a solution but most probably not the best one. We want to have some kind of social login implemented sooner rather than later and would try to store state outside of the app and browser (cookie).
So, lets keep this open for now and thanks for the feedback.

**Comment by VanderLeal** [comment]
A social login with "cloud based" preferences is 10x better, say no more. :)

btw, if you need a test user for colour blind I have one playing with me, and he's playing 6 concurrent games atm.
So, surely a good source for (colour) blind tests... 😅

Thanks!

**Comment by kevinb9n** [comment]
Ranked preferences for color would be nice, but I would argue against a change that shows the same game to different people with different color assignments. Players in chat will say things like "yeah I had to place there even tho it gave red a point" and people will be confused. I'm sure that wouldn't happen *often* but I just feel like the game should look the same to everyone.
